interface/weno/confirm.php

```diff
@@ -17,13 +17,13 @@
 
 use OpenEMR\Common\Csrf\CsrfUtils;
```

Comment:
This is a stupid question, but where is this confirm.php script called? I wasn't able to find it.

Reply:
Also, if this script is not used. Does that mean the interface/weno/jsonScript.php is no longer used?
(which also appears to be using the nonexistent Rx\Weno\TransmitData class)

Reply:
These are the old scripts and they can be removed from the system. I was going to wait but I can remove them now. 
I removed the processing of the JSON from the browser and this can be removed now also.